src/aere_parse.erl

```diff
@@ -3,10 +3,9 @@
 -export([ parse/1, words/1, commands/0, resolve_command/1 ]).
 
 -type parse_result() :: {atom(), string() | [string()]}
-                      | no_return()
-                      | skip.
```

Comment:
@radrow Why have you decided to remove `skip` from the `parse_result` type even though the `parse` function returns `{skip, []}`?

Reply:
Because skip /= {skip, []}

Reply:
Yeah, now I see that `{atom(), string() | [string()]}` fits `{skip, []}` as well. I missed that one.